bilibili/dist/metadata.json

```diff
@@ -24,17 +24,17 @@
     "game.bilibili.com",
     "show.bilibili.com",
     "member.bilibili.com",
-    "account.bilibili.com",
-    "biligame.com"
+    "message.bilibili.com",
+    "search.bilibili.com",
+    "account.bilibili.com"
   ],
-  "version": "1.8",
+  "version": "1.9.0",
   "logo": "https://i.imgur.com/w7q6IS0.png",
   "thumbnail": "https://i.imgur.com/wKnEEei.jpg",
   "color": "#05acff",
   "tags": [
     "video",
     "anime"
   ],
-  "category": "anime",
-  "regExp": "(([a-z0-9]+)[.])?bilibili[.]com"
+  "category": "anime"
```

Comment:
add this regex and biligame should also work. 
`(([a-z0-9]+)[.])?(bilibili|biligame)[.]com`